portal-wire-protocol.md

```diff
@@ -232,13 +232,20 @@ Signals interest in receiving the offered data from the corresponding Offer mess
 
 ```
 selector     = 0x07
-accept       = Container(connection_id: Bytes2, content_keys: BitList[limit=64]]
+accept       = Container(connection_id: Bytes2, content_keys: ByteList[limit=64])
 ```
 
 - `connection_id`: Connection ID to set up a uTP stream to transmit the requested data.
     - ConnectionID values **SHOULD** be randomly generated.
 - `content_keys`: Signals which content keys are desired.
-    - A bit-list corresponding to the offered keys with the bits in the positions of the desired keys set to `1`.
+    - A byte-list corresponding to the offered keys with the byte in the positions of the desired keys set to `1`.
+      - 0: Generic decline, catch all if their is no specified case
+      - 1: Accept the Content
```

Comment:
Minor remark, but I would be in favor of putting the accept on the 0, so that all declines come after that in order.

Reply:
Another remark would be if the generic decline is needed?

Reply:
> Minor remark, but I would be in favor of putting the accept on the 0, so that all declines come after that in order.

Awesome I wanted to do that like posix error's but I didn't know if team's were ok with initializing the bitlist with 1's or not



> Another remark would be if the generic decline is needed?

There are possibly cases I didn't think of and we could in the future update the spec to handle them, I thought it was a good catch all, as posix/linux uses 1 for a catch all. I am not too picky though so if you really want to remove it I am cool with that just let me know

Reply:
> There are possibly cases I didn't think of and we could in the future update the spec to handle them, I thought it was a good catch all, as posix/linux uses 1 for a catch all. I am not too picky though so if you really want to remove it I am cool with that just let me know

I'm fine with keeping the generic case also. I just mentioned it as I looked in fluffy code and in our case we have no condition to use it currently.

Reply:
We don't use it as well. My thinking was if there wasn't any specific errors for a client they would use it, then we could ask them why they are using a generic code and go from there